client/landing/jetpack-cloud/routes.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import {
+	clientRender,
+	jetpackCloud,
+	jetpackBackups,
+	jetpackScan,
+	makeLayout,
+	setupSidebar,
+} from './controller';
+
+const router = ( baseRoute = '' ) => {
+	page( baseRoute + '/', setupSidebar, jetpackCloud, makeLayout, clientRender );
+
+	page( baseRoute + '/backups', setupSidebar, jetpackBackups, makeLayout, clientRender );
```

Comment:
Should we have custom config flags for `scan` and `backup`, and enable those sections only if those config flags are enabled?